admin/controllers/pupil-pin.js

```diff
@@ -104,6 +106,11 @@ const postGeneratePins = async (req, res, next) => {
     if (update) {
       await schoolDataService.sqlUpdate(R.assoc('id', school.id, update))
     }
+
+    // New code - writes to allocateCheckFormTable, depends on school pin being ready
+    // disabled as not yet working correctly in travis.
+    // await checkStartService.prepareCheck2(pupilsList, req.user.School, req.user.schoolId, pinEnv === 'live')
```

Comment:
Should the new code not be running now, for this feature to be complete? I was expecting this to be uncommented and replace the old code above on line 98?

Reply:
see above